### PR TITLE
generate aircraft homepage

### PR DESCRIPTION
**Pull request by Zhenni-Z**
# REMOVE THIS AND ADD A TITLE HERE

## Description

Created aircraft homepage

## Types of change

- [x] New feature (change which adds functionality)
- [ ] Bug fix (change which fixes an issue)

- [ ] Improvement/refactoring (change that doesn't add any features but makes things better)

- [ ] Styling (style changes that don't affect functionality)

## Checklist before submitting

- [x] Satisfies card on Trello

- [ ] Has had a another team mate review your design.

- [ ] No known defects

- [ ] Appropriate tests completed (e.g. core functionality, edge cases)

- [ ] Code has been formatted with nicely and contains proper indentation.

- [ ] Code comments as appropriate

- [ ] `master` branch has been merged into the feature branch and tested (if any newer commits on `master`)

### CODE REVIEW

File: app/views/aircrafts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+              <div class="flex align-items-center ms-3">
+                <i class="fa-solid fa-euro-sign"></i>
+                <span><%= aircraft.arrival_address %></span>
+              </div>
+            </div>
+          </div>
+          <div class="flat-card-actions">
+            <%= link_to "More info", aircraft_path(aircraft) %>
+            <%= link_to "Edit", edit_aircraft_path(aircraft), class: "ms-3 text-warning" %>
+            <%= link_to "Remove", aircraft_path(aircraft),
```
Comment (learner1unknown) [inline app/views/aircrafts/index.html.erb:31]: User should not be able to edit or remove any aircraft from hompage, please remove unecessary links to actions. Also remove the data actions.

File: app/views/aircrafts/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+  </div>
+
+  <div class="row">
+    <% @aircrafts.each do |aircraft| %>
+      <div class="col-md-4">
+        <div class="aircraft-card">
+          <div class="aircraft-card-content">
+            <div class="aircraft-card-title">
+              <h4><%= aircraft.description %></h4>
```
Comment (learner1unknown) [inline app/views/aircrafts/index.html.erb:14]: The heading to display is not description but title - refer to schema

File: app/views/aircrafts/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            </div>
+            <p><%= aircraft.aircraft_type %></p>
+            <div class="flex">
+              <div class="flex align-items-center">
+                <i class="fa-solid fa-people-line"></i>
+                <span><%= aircraft.departure_address %></span>
+              </div>
+              <div class="flex align-items-center ms-3">
+                <i class="fa-solid fa-euro-sign"></i>
```
Comment (learner1unknown) [inline app/views/aircrafts/index.html.erb:23]: why are there euro dollar signs here ?

File: app/views/aircrafts/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      <div class="col-md-4">
+        <div class="aircraft-card">
+          <div class="aircraft-card-content">
+            <div class="aircraft-card-title">
+              <h4><%= aircraft.description %></h4>
+            </div>
+            <p><%= aircraft.aircraft_type %></p>
+            <div class="flex">
+              <div class="flex align-items-center">
+                <i class="fa-solid fa-people-line"></i>
```
Comment (learner1unknown) [inline app/views/aircrafts/index.html.erb:19]: why is there a people image here ?

File: app/views/aircrafts/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,40 @@
+<div class="container mt-3">
+  <div class="row">
+    <div class="col-12">
+      <h1>All our Aircrafts</h1>
```
Comment (learner1unknown) [inline app/views/aircrafts/index.html.erb:4]: title can just be "Fly for a day or two?" or "All Aircraft" "our aircraft" doesn't make sense

File: config/routes.rb [hunk 6]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "pages#home"
```
Comment (learner1unknown) [inline config/routes.rb:3]: Update root to "aircrafts#home" so that the root also goes to the hompage

**Review by learner1unknown — CHANGES_REQUESTED** [review]
Good job!, when copying and pasting code have a read of it and adapt it to your use case. 
Refer to my inline comments.
